scripts/common/object_store.sh

```diff
@@ -157,20 +154,19 @@ spec:
     command:
     - /usr/local/bin/kurl
     - sync-object-store
-    - --source_host=$RGW_HOST
-    - --source_access_key_id=$RGW_ACCESS_KEY_ID
-    - --source_access_key_secret=$RGW_ACCESS_KEY_SECRET
-    - --dest_host=$MINIO_HOST
-    - --dest_access_key_id=$MINIO_ACCESS_KEY_ID
-    - --dest_access_key_secret=$MINIO_ACCESS_KEY_SECRET
+    - --source_host=$source_host
+    - --source_access_key_id=$source_access_key
+    - --source_access_key_secret=$source_secret_key
+    - --dest_host=$destination_host
+    - --dest_access_key_id=$destination_access_key
+    - --dest_access_key_secret=$destination_secret_key
```

Comment:
I think we already have a function that copies object stores [here](https://github.com/replicatedhq/kURL/blob/main/scripts/common/object_store.sh#L69-L72). Maybe we can replace this with a call to `migrate_object_store`.

Reply:
They're not quiiiiiite the same, but I'll give it a go

Reply:
easy enough